packages/utils/mocks/fixtures/execute-progress.mock.mjs

```diff
@@ -1,5 +1,5 @@
 import { bold, gray } from 'ansis';
-import { getProgressBar } from '../../../../dist/packages/utils/index.js';
+import { getProgressBar } from '../../../../dist/packages/utils/index.js.js';
```

Comment:
Double-extension here. The `index.js` can actually be omitted since there's a `package.json`.

```suggestion
import { getProgressBar } from '../../../../dist/packages/utils';
```